src/validator/validator.service.js

```diff
@@ -9,7 +9,7 @@ export default function () {
   this.MAX_DOMAIN_LENGTH = 63;
 
   this.isValidIpv4 = function isValidIpv4(ip) {
-    if (ip != null && _.isString(ip) && ip.split('.').length === 4) {
+    if (ip != null && isString(ip) && ip.split('.').length === 4) {
```

Comment:
```suggestion
    if (isString(ip) && ip.split('.').length === 4) {
```

is enough 😅 

(I know you are not supposed to fix this, don't do it)